src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -45,27 +45,27 @@ public partial class Process : System.ComponentModel.Component, System.IDisposab
         public System.IntPtr MaxWorkingSet { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd")] set { } }
         public System.IntPtr MinWorkingSet { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")] [System.Runtime.Versioning.SupportedOSPlatformAttribute("freebsd")] set { } }
         public System.Diagnostics.ProcessModuleCollection Modules { get { throw null; } }
-        [System.ObsoleteAttribute("This property has been deprecated.  Please use System.Diagnostics.Process.NonpagedSystemMemorySize64 instead.  https://go.microsoft.com/fwlink/?linkid=14202")]
+        [System.ObsoleteAttribute("This property has been deprecated and may return incorrect results. Please use System.Diagnostics.Process.NonpagedSystemMemorySize64 instead.  https://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
"and may return incorrect results"... should we say why? e.g. "and may return incorrect results due to large values that overflow the Int32 type" or something like that?

Reply:
I could, but it doesn't seem worth it because I don't think there's ever a reason to use it. I don't think it would be reasonable for folks to say "Hmm, OK given that info I'll use it because I'm confident I'm only going to examine processes that don't use that much memory and I'm going to avoid a cast". I think it's fair to claim that in almost any reasonable circumstances, it could give incorrect results. Even IoT devices nowadays can have GB's of memory.

Reply:
I'm not saying there's a reason to use it.  But if that error is displayed, it could easily be interpreted as "this method has a bug we didn't want to fix because we're lazy so we simply deprecated it".

Reply:
I am not concerned about that, but I can change it if you like, do you want me to?

Reply:
The error is only displayed if someone is using it, and it'd be nice if when we say "and may return incorrect results" we provide a bit more clarity as to when/why so that it's clear why it was deprecated.  So, yes, please.  I'd also be fine if we say something like "This property has been deprecated because the type of the property can't represent all valid results."  Up to you.

Reply:
Yeah that's an improvement